grantpicks/api.py

```
@@ -0,0 +1,400 @@
+                            "total_fund_received": 87.2,
+                            "rounds_participated": 2,
+                            "total_votes": 2,
+                        },
+                        response_only=True,
+                    ),
+                ],
+            ),
+            500: OpenApiResponse(description="Internal server error"),
+        }
+    )
+    def get(self, request: Request, *args, **kwargs):
```

Ruff (0.14.1)

Unused method argument: `request`

(ARG002)